index.d.ts

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+
+type DateInfo = {
+    day: number;
+    month: number;
+    year: number;
+}
+
+type Range = {
+    from: DateInfo | null;
+    To: DateInfo | null;
+}
+
+export interface DatePickerProps {
+    value: Range | DateInfo[] | null;
+    onChange: (newValue: Range | DateInfo[] | null) => null;
+    formatInputText?: () => string;
+    inputPlaceholder?: string;
+    inputClassName?: string;
+    renderInput?: ({ ref, onFocus, onBlur }: { ref: React.RefObject<HTMLInputElement>, onFocus: () => void, onBlur: () => void }) => React.ReactElement;
+    wrapperClassName?: string;
+    calendarClassName?: string;
+    calendarTodayClassName?: string;
+    calendarSelectedDayClassName?: string;
+    calendarRangeStartClassName?: string;
+    calendarRangeBetweenClassName?: string;
+    calendarRangeEndClassName?: string;
+    disabledDays?: number[];
+    onDisabledDayError?: (number) => null;
+    colorPrimary?: string;
+    colorPrimaryLight?: string;
+    minimumDate?: DateInfo | null;
+    maximumDate?: DateInfo | null;

```

Comment:
As it's optional there is no need to `null`.

Reply:
Inside component default value of these two is null,so i have to consider it 

![image](https://user-images.githubusercontent.com/35793921/68698798-f745c700-0596-11ea-8aaf-1ccaee8f0f55.png)

Reply:
It's the library business. Developers have two options; They can define the `minimumDate`/`maximumDate` or leave them undefined.